src/main/java/ch/njol/skript/conditions/CondIsBlocking.java

```diff
@@ -31,15 +31,15 @@
  * @author Peter Güttinger
  */
 @Name("Is Blocking")
-@Description("Checks whether a player is blocking with his shield.")
+@Description("Checks whether a player is blocking with his shield.")

```

Comment:
While you're on it, it'd be good to change this description to "... blocking with **their** shield." No reason for this to be gendered.